examples/showcase/tracked-controls/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Room Scale</title>
+    <meta name="description" content="Room Scale – A-Frame">
+    <script src="../../../dist/aframe.js"></script>
+    <script src="js/vendor/BlendCharacter.js"></script>
+    <script src="js/components/blend-model.js"></script>
```

Comment:
Putting the components/shaders in the JS directory rather than the root of the example might make them less discoverable. `js/components/` vs `components/`